boards.txt

```diff
@@ -532,6 +571,14 @@ Maple.menu.xserial.none.build.xSerial=-DHAL_UART_MODULE_ENABLED -DHWSERIAL_NONE
 Maple.menu.xserial.disabled=Disabled (No Serial)
 Maple.menu.xserial.disabled.build.xSerial=
 
+RemRam.menu.xserial.generic=Generic Serial
```

Comment:
It seems your a not on the latest core.
You should rebase those menu has been review on 1.3.0.